### ISSUE TITLE
Add Social Sharing to key content pages

### ISSUE THREAD

**Issue by lyndoer** [massgov/feature-requests#178]
Please add social share buttons to key content pages.  My guess would be that helpful content types for sharing would be forms, events, and news.

**Comment by shaed333** [comment]
Hello @lyndoer, Thank you for your feature request. Can you tell us a little bit more about why you think that forms is an important content type to have the social share?

**Comment by lyndoer** [comment]
I just offered that as an example of what I think would make a good content type for social sharing but most of my agency's forms are built and maintained by 3rd parties.

I would imagine that, if there was a form for, as an example, submitting a consumer complaint to the Attorney General's office, that an advocate who is trying to help several people with a particular consumer issue could post it to their organization's Facebook or Twitter page and get it out to several people at once.  

For example, someone could post the consumer complaint form for people still holding L'Espalier gift cards if they have a community of Mass food lovers (https://www.bostonglobe.com/business/2019/01/22/upscale-restaurant-closing-leaves-holders-gift-cards-hungry-for-refunds/ze2dje1VQxHZMD0j2xaUnO/story.html).

**Comment by Lchiong** [comment]
Agreed--I think this would be a helpful feature for the OIG as well. We are building up our public resources content, and it would be helpful to be able to share it on social (such as news items).
